Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -13,6 +13,14 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
     fileprivate(set) var actions: [[PhotonActionSheetItem]]
 
     var syncManager: SyncManager? // used to display the sync button
+    
+    enum ButtonToastAction {
+        case share
+        case addToReadingList
+        case bookmarkPage
+        case removeBookmark
+        case copyUrl
+    }
```

Comment:
Is there a better place for this than in PhotonActionSheet?

Reply:
Maybe it belongs better in `PageActionMenu` since it is relating more to that than the PhotonActionSheet